docker-compose-generator/docker-fragments/bgold-lnd.yml

```diff
@@ -31,21 +31,11 @@ services:
 
   btcpayserver:
     environment:
-      BTCPAY_BTCLIGHTNING: "type=lnd-rest;server=http://lnd_bitcoingold:8080/;macaroonfilepath=/etc/lnd_bitcoingold/admin.macaroon;allowinsecure=true"
+      BTCPAY_BTGLIGHTNING: "type=lnd-rest;server=http://lnd_bitcoingold:8080/;macaroonfilepath=/etc/lnd_bitcoingold/admin.macaroon;allowinsecure=true"
```

Comment:
I'm not familiar with the codebase, but this looks like a typo (`BTG`). Ignore if not.

Reply:
this was actually a bug with some altcoin misconfiguring stuff if activated

Reply:
Ohh wait, bitcoingold... Nevermind.